chain/chain/src/chain.rs

```diff
@@ -203,6 +206,8 @@ pub struct Chain {
     /// Block economics, relevant to changes when new block must be produced.
     pub block_economics_config: BlockEconomicsConfig,
     pub doomslug_threshold_mode: DoomslugThresholdMode,
+    #[cfg(feature = "sandbox")]
+    pub pending_states_to_patch: Option<Vec<StateRecord>>,
```

Comment:
```suggestion
    pending_states_to_patch: Option<Vec<StateRecord>>,
```

`pub` means that the item is visible from outside of the crate. So, the absence of `pub` helps with understanding the code a lot, because, at a glance, you immediately know that anything that happens to an item happens in the current crate.